ci/infra/testrunner/README.md

```diff
@@ -1,63 +1,213 @@
-
 # Testrunner
 
-## Local Dev Machine Setup
-Set the following environment variables
+## Contents
+
+- [Summary](#summary)
+- [Configuration](#configuration-parameters)
+  - [Work Environment](#work-environment)
+  - [Platform](#platform)
+    - [Terraform](#terraform)
+    - [Openstack](#openstack)
+    - [VMware](#vmware)
+  - [Skuba](#skuba)
+  - [Log](#log)
+  - [Test](#test)
+- [Environment setup](#environment-setup)
+  - [Local setup](#local-setup)
+  - [CI setup](#ci-setup)
+- [Usage](#usage)
+  - [General CLI options](#general-cli-options)
+  - [Provision](#provision)
+  - [Node commands](#node-commands)
+    - [Node Upgrade](#node-upgrade)
+  - [Ssh](#ssh)
+  - [Test](#ssh)
+- [Examples](#examples)
+  - [Create K8s Cluster](#create-k8s-cluster)
+  - [Collect logs](#collect-logs)
+
+## Summary
+
+Testrunner is a CLI tool for setting up a running e2e tests, making transparent the mechanism used for providing the test infrastructure. It can used as a stand-alone tool, running either locally o as part of a ci pipeline. It provides commands for provisioning the infrastructure, deploying a k8s cluster using `skuba` and running tests. It also provides [a library](tests/README.md) for developing `pytest`-based tests.
```

Comment:
```suggestion
Testrunner is a CLI tool for setting up an environment for running e2e tests, making transparent the mechanism used for providing the test infrastructure. It can used as a stand-alone tool, running either locally or as part of a ci pipeline. It provides commands for provisioning the infrastructure, deploying a k8s cluster using `skuba` and running tests. It also provides [a library](tests/README.md) for developing `pytest`-based tests.
```